research/fed-ce/requirements.txt

```diff
@@ -0,0 +1,8 @@
+nvflare>=2.3.0
+websockets
```

Comment:
websockets should be installed by nvflare, I think.

Reply:
thanks! removed (was for last version)